### PR TITLE
MON-11764 sudo perms

### PR DESCRIPTION
**Pull request by sjoegren**
Most of the diff is just EOL space change created by the `git add --renormalize` command. Can be ignored with `git diff master --ignore-space-at-eol`.

### CODE REVIEW

File: system/views/kohana_profiler_table.css [hunk 1]
```diff
@@ -1,53 +1,53 @@
...
+}
+#kohana-profiler .kp-table .kp-name
+{
+	width: 200px;
+	border-right: 1px solid #E5EFF8;
+}
+#kohana-profiler .kp-table .kp-altrow .kp-data, #kohana-profiler .kp-table .kp-altrow .kp-name
+{
+	background-color: #F6F8FB;
 }
```
Comment (AxelBolle) [inline system/views/kohana_profiler_table.css:53]: Omg, whitespace missing

**Review by AxelBolle — APPROVED** [review]
LGTM

File: system/views/kohana_profiler_table.css (hunk 1 above)
Comment (sjoegren) [reply system/views/kohana_profiler_table.css:53]: Hm, I though that git's EOL normalization would've fixed this too, but apparently not. I'll fix it manually.

Comment (sjoegren) [reply system/views/kohana_profiler_table.css]: Fixed in fixup commit. Will squash them now.

**Comment by jdumalaonITRS** [comment]
rebasing to get builds for verification

**Pull request merged**